Clean-up/Terraform/main.tf

```
@@ -0,0 +1,55 @@
+# This file terraforms lambda and eventbridge on AWS
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.AWS_REGION
+}
+
+# terraform import aws_lambda_function.c11-hermes-clean_lambda c11-hermes-clean_lambda
+resource "aws_lambda_function" "c11-hermes-clean_lambda" {
+    function_name                  = "c11-hermes-clean_lambda"
+    image_uri                      = "129033205317.dkr.ecr.eu-west-2.amazonaws.com/c11-hermes-clean@sha256:e9ccc95322d51a339b97f61fc1f109b1f9f0d21a4598c3d470c6f857328587ca"
+    package_type                   = "Image"
+    role                           = "arn:aws:iam::129033205317:role/service-role/c11-hermes-clean_lambda-role-yk4tvt6d"
```

Should this role be created in Terraform?

---

I'll add it now

---

Consider changing this to:
```suggestion
    role                           = aws_iam_role.lambda_role.arn
```

---

Added